posthog/models/dashboard_templates.py

```diff
@@ -25,6 +25,7 @@ class DashboardTemplate(UUIDModel):
     created_by: models.ForeignKey = models.ForeignKey("User", on_delete=models.SET_NULL, null=True, blank=True)
     deleted: models.BooleanField = models.BooleanField(blank=True, null=True)
     image_url: models.CharField = models.CharField(max_length=8201, null=True, blank=True)
+    is_public: models.BooleanField = models.BooleanField(blank=True, null=True)
```

Comment:
If we use a boolean flag then we only have the two states everyone and only the owning team of the instance

Should we have a scope...? Starts as "PUBLIC" or "TEAM" but allows extension to include e.g. "ORGANIZATION"

Reply:
Yep makes a lot of sense, will switch it